server/routes/user.tsx

```diff
@@ -6,12 +6,22 @@ import { getUser } from 'server/utils/queryHelpers';
 import { handleErrors } from 'server/utils/errors';
 import { getInitialData } from 'server/utils/initData';
 import { generateMetaComponents, renderToNodeStream } from 'server/utils/ssr';
+import { getCountOfPubsOrCollections } from 'server/community/queries';
+// check if member has any pubs or collections for user
```

Comment:
This comment has come detached from the function it describes — probably best to remove it.